src/libs/OptionsListUtils.ts

```diff
@@ -99,6 +99,7 @@ type GetOptionsConfig = {
     includePolicyTaxRates?: boolean;
     policyTaxRates?: PolicyTaxRateWithDefault;
     transactionViolations?: OnyxCollection<TransactionViolation[]>;
+    actionTypeForParticipants?: string;
```

Comment:
`type ActionType = ValueOf<typeof CONST.REPORT.TYPE | typeof CONST.IOU.TYPE | typeof CONST.IOU.REQUEST_TYPE>;`
```suggestion
    actionTypeForParticipants?: ActionType;
```